sql/moz-fx-data-shared-prod/cloudflare_derived/device_usage_v1/metadata.yaml

```diff
@@ -6,6 +6,10 @@ owners:
 labels:
   incremental: false
   owner1: kwindau@mozilla.com
+scheduling:
+  dag_name: bqetl_cloudflare
+  date_partition_parameter: dte
+  arguments: ["--date", "{{ds}}", "--cloudflare_api_token", "{{ var.value.cloudflare_auth_token}}"]
```

Comment:
It's unfortunately not recommended to use Airflow env variables for tokens or any secrets. Instead `secrets` should be used: https://mozilla-hub.atlassian.net/wiki/spaces/SRE/pages/27922811/WTMO+Developer+Guide#WTMODeveloperGuide-ManagingsecretsinGKEworkloads

I don't think `secrets` are currently supported in the DAG generation logic, but it seems worth adding

Reply:
I created a ticket here: https://github.com/mozilla/bigquery-etl/issues/5778

Reply:
@scholtzan - do you know any existing DAG that uses this secrets method that I can use as an example?